easybuild/easyconfigs/m/Mercurial/Mercurial-5.6.1-foss-2020b.eb

```diff
@@ -0,0 +1,33 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = "PythonPackage"
+
+name = 'Mercurial'
+version = '5.6.1'
```

Comment:
Is there a specific need for both 5.6.1 and 5.7.1? If so then the comments for 5.7.1 apply here as well.

Reply:
Originally, I had only 5.6.1 that I created some time ago. I checked if there a new release and it was so I added the newest one.  I can remove 5.6.1 since there is no significant difference between the two and this package wasn't updated for a long time. In other words, Mercurial is not really used as much as for instance git.